docs/releases/upgrading/upgrade-v10.0.md

```diff
@@ -0,0 +1,110 @@
+# Upgrading from v9.0 to v10.0
+
+## Prerequisites
+
+The steps listed in this article require an existing local installation of InvenioRDM v9.0, please make sure that this is given!
```

Comment:
v9.0 or v9.1? when instally `invenio-cli init -c v9` (or current installation) it pulls v9.1 if I am correct. I think there is no changes in upgrade from .0 to .1. But I'd avoid specifying minor then.

```suggestion
The steps listed in this article require an existing local installation of InvenioRDM v9, please make sure that this is given!
```